### ISSUE TITLE
Ошибка с \Bitrix\Main\Entity\ExpressionField

### ISSUE THREAD

**Issue by FaritSlv** [StanislavDremin/ab.iblock#2]
Доброго времени суток. 
Изучаю Ваше решение. На данный момент нашел один недочет. 
При работе с \AB\Iblock\Element, если в select добавить \Bitrix\Main\Entity\ExpressionField вылетает ошибка в методе \AB\Iblock\Manager::prepareOnValues из-за array_unique. 
Получается, что \AB\Iblock\Element не полностью поддерживает \Bitrix\Main\Entity\DataManager.
Пример:
`Element::getList(array(
                'select' => array(
                    'PROPERTY.PRICE',
                    new \Bitrix\Main\Entity\ExpressionField('MIN_ID', 'MIN(%s)', array('ID'))
                ),
                'runtime' => array(
                    new \Bitrix\Main\Entity\ReferenceField(
                        'SECTION',
                        \Bitrix\Iblock\SectionTable::getEntity(),
                        array(
                            '=ref.ID' => 'this.IBLOCK_SECTION_ID'
                        )
                    )
                ),
                'filter' => array(
                    "SECTION.ID" => array(8, 10, 12)
                ),
                'group' => array(
                    "PROPERTY.PRICE"
                )
            ));`

**Comment by gromdron** [comment]
Да, есть такая проблема на приведении типов. Воспроизвел и получил ожидаемый fatal на приведение объекта к строке.
#3
